src/service_daemon.rs

```diff
@@ -1344,7 +1344,7 @@ impl Zeroconf {
         if info.is_addr_auto() {
             let selected_addrs = self.selected_addrs(my_ip_interfaces(true));
             for addr in selected_addrs {
-                info.insert_ipaddr(addr);
+                info.insert_scoped_addr(addr, None);
```

Comment:
I'm not sure yet if we can eliminate this None and the one in https://github.com/keepsimple1/mdns-sd/pull/342/files#diff-2061a30a0f84a30fe5c20cfb9505c70e8efd1147fc27ebb4deb5e35c53d3a59fR205 which would make the Option superfluous.